examples/multiple-namespaces/mud.config.ts

```diff
@@ -1,23 +1,53 @@
 import { defineWorld } from "@latticexyz/world";
 
 export default defineWorld({
-  namespace: "game",
-  codegen: { namespaceDirectories: true },
-  tables: {
-    Health: {
-      schema: {
-        player: "address",
-        value: "uint32",
+  namespaces: {
+    game: {
+      tables: {
+        Position: {
+          schema: {
+            player: "address",
+            x: "int32",
+            y: "int32",
+          },
+          key: ["player"],
+        },
+        Victory: {
+          schema: {
+            value: "bool",
+          },
+          key: [],
+        },
       },
-      key: ["player"],
     },
-    Position: {
-      schema: {
-        player: "address",
-        x: "int32",
-        y: "int32",
+    somePlugin: {
+      tables: {
+        Score: {
+          schema: {
+            player: "address",
+            score: "uint256",
+          },
+          key: ["player"],
+        },
+      },
+    },
+    gameFork: {
```

Comment:
it's not immediately obvious to me what this example is supposed to communicate

I wonder if there's another example we can use here that is more self explanatory

Reply:
oh whoops, I was confusing this with the changeset! I think this is fine as an example app, just wanted to make sure the example config in the changeset was intuitive/self-explanatory

Reply:
> The hacker namespace demonstrates access control by attempting to write to a table in game

I think this might not be clear and the name itself could be misleading

Maybe we can use tokens as a better example? where we might have many of the same tables across namespaces

Reply:
Just removed the `hacker__` example, we don't need to demonstrate access control here